src/standalone.js

```diff
@@ -13,5 +13,8 @@ governing permissions and limitations under the License.
 // This file is used by rollup to create the browser version that is uploaded to cdn
 
 import core from "./core/index.js";
+import * as components from "./core/componentCreators.js";
 
-core();
+// If you change this line, check if the custom build script is still working.
+// You might need to change the babel plugin in scripts/helpers/entryPointGeneratorBabelPlugin.js.
+core({ components });
```

Comment:
`generateInputEntryFile` method from `alloyBuild.js` script, will read this file and transform it using Babel to use just the selected optional components that the user chooses. For example, if a user wants custom build with Privacy and Personalization components, this file will be transformed to a custom entry file with the following content:

```
import core from "./core/index.js";
import * as components from "./core/componentCreators.js";

core({
  components: {
    privacy: components.privacy,
    personalization: components.personalization,
  },
});
```

This will become the entry point file used by rollup. Tree shake will work.